### ISSUE TITLE
QA: Send all emails from MIT domain

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1358]
## Summary

We recently found out that we need to change the way that we handle emails from researchers to study participants (via the study's 'contact participants' page) in order to meet new email deliverability requirements. Emails sent from researchers via the Contact Participants page are now sent from the lookit MIT email address (same address as the automated emails), with the lab email in the "reply-to" field instead of "from". This change is now live on the staging site.

## QA

Here are the things to check on staging:

### 1. Custom messages sent from a researcher/study to participant

  - From a researcher account, find a study or create one if necessary (see below for an example of a minimal JSON protocol)
  - Participate in the study. Ideally this would be from an account that uses a Gmail username. It can be the same as your researcher account or a separate participant-only account.
  - From your researcher account, 'accept' the consent for your response.
  - Go to your study details page -> Contact Participants, and send a message to yourself.
  - **You should receive the message in an email sent from lookit-staging-bot@mit.edu** (NOT the contact email for the lab that your study is associated with).
  - Click 'reply' on the message that you received. **The email address that appears in the 'to' field for your reply should match the contact email for the lab that your study is associated with**. E.g. if your study is part of the 'demo' lab then the reply-to address should be lookit+demo@mit.edu.
  - If you have access to the lab's email account, you can try replying to the email and check that it is received by the lab email account.

### 2. Other emails that should be affected by this change

All of the emails types below were previously sent from a lab email address and should now be sent from lookit-staging-bot@mit.edu. We should check at least a few of these to make sure that (1) the email is delivered in response to the action, and (2) the email is sent from the MIT address instead of the lab address. (These emails do not include the lab contact as the 'reply-to' address, so no need to check that.)
  - Notification sent to a researcher that they have been added to a lab as a guest
  - Notification sent to a researcher that they have been added to a lab as a member
  - Notification sent to a researcher that they are now an admin of a lab
  - Notification sent to a researcher that they have been removed from a lab
  - Notification sent to a researcher that they have been granted access to a particular study

### 3. Other emails that should NOT be affected by this change

The following types of emails should work the same as before (i.e. still sent from lookit-staging-bot email address and no separate reply-to address). We should check at least a few of these to make sure that they're still working.
  - Notification sent to researcher that frame data dictionary has been created (in a study that contains response data, go to Study Responses -> All Responses -> Frame data -> "Data dictionary (CSV)" button).
  - Notification sent to researcher that video archive has been created (in a study that contains response data, go to Study Responses -> Videos -> "Download all videos" button).
  - Study announcements
  - Notification sent to researcher that experiment was built (or failed to build)
  - Notification sent to researcher that study was approved / changes requested / approval rescinded
  - Notification sent to lab admins that a researcher has requested to join the lab
  - Notification sent to lab admins that their lab has been approved
  - Notification sent to Lookit site admins that lab has been created
  - Notification sent to Lookit site admins that a study has been submitted for approval
  - Notification sent to Lookit site admins that a study has been retracted
  - Notification sent to Lookit site admins that a study has been activated
  - Notification sent to Lookit site admins that a study has been paused
  - Notification sent to Lookit site admins that a study has been deactivated

## Minimal JSON protocol for a test study

```json
{
    "frames": {
        "video-consent": {
            "kind": "exp-lookit-video-consent",
            "template": "consent_005",
            "PIName": "Jane Smith",
            "institution": "Science University",
            "PIContact": "Jane Smith at 123 456 7890",
            "purpose": "Why do babies love cats? This study will help us find out whether babies love cats because of their soft fur or their twitchy tails.",
            "procedures": "Your child will be shown pictures of lots of different cats, along with noises that cats make like meowing and purring. We are interested in which pictures and sounds make your child smile. We will ask you (the parent) to turn around to avoid influencing your child's responses.",
            "payment": "After you finish the study, we will email you a $5 BabyStore gift card within approximately three days. To be eligible for the gift card your child must be in the age range for this study, you need to submit a valid consent statement, and we need to see that there is a child with you. But we will send a gift card even if you do not finish the whole study or we are not able to use your child's data! There are no other direct benefits to you or your child from participating, but we hope you will enjoy the experience.",
            "datause": "We are primarily interested in your child's emotional reactions to the images and sounds. A research assistant will watch your video to measure the precise amount of delight in your child's face as he or she sees each cat picture.",
            "research_rights_statement": "You are not waiving any legal claims, rights or remedies because of your participation in this research study.  If you feel you have been treated unfairly, or you have questions regarding your rights as a research subject, you may contact the [IRB NAME], [INSTITUTION], [ADDRESS/CONTACT]"
        },
        "exit-survey": {
            "kind": "exp-lookit-exit-survey",
            "debriefing": {
                "title": "Thank you!",
                "text": "PLACEHOLDER DEBRIEFING TEXT. SEE DOCS FOR ADVICE ON WHAT TO INCLUDE HERE."
            }
        }
    },
    "sequence": [
        "video-consent",
        "exit-survey"
    ]
}

```

**Comment by becky-gilbert** [comment]
I tested the first item here (Custom messages sent from a researcher/study to participant) on staging and found that everything is working as it should be. The email was sent from the lookit bot email address, with the lab email as the reply-to address. The only thing I wasn't able to check was that participant replies were actually sent to and received by the lab email address, because I don't have access to any of the lab email accounts. Is someone else able to check this?

I can also confirm that I tested a few emails in the third category above (emails that should NOT be affected by this change), such as study status change and experiment build messages, and these are still working fine.

**Comment by ianchandlercampbell** [comment]
Seems to be working as intended! My account email and lab email are the same so Gmail bundled them. Can do additional testing with a second email in a bit.



![Image](https://github.com/lookit/lookit-api/assets/75502403/f4a6f532-7c85-4405-9ba7-28ddd4c5a520)

**Comment by becky-gilbert** [comment]
Closed via #1357 and #1366
